ddp-workspace/projects/ddp-dsm-ui/src/app/navigation/navigation.component.html

```diff
@@ -49,6 +49,10 @@
           <!-- Setup label dimensions, only if you have the right to handle samples -->
           <hr *ngIf="hasRole().allowedToHandleSamples()">
           <li *ngIf="hasRole().allowedToHandleSamples()"><a [routerLink]="['labelSettings']"> Label Settings </a></li>
+          <hr *ngIf="hasRole().allowedToHandleSamples()">
+          <li *ngIf="hasRole().allowedToHandleSamples() || hasRole().allowedToDoOrderSequencing()">
+            <a [routerLink]="['clinicalOrders']">Clinical Orders</a>
```

Comment:
are we really calling that Clinical Orders?

Reply:
This is from the instructions in the doc Taylor has made